app/views/va_common/_footer.html.erb

```diff
@@ -1,61 +1,54 @@
     </div>
   </div>
 </div>
 
-
-<div class="footer" role="contentinfo">
+<footer class="footer" role="contentinfo">
   <div class="row wow fadeIn animated">
     <div class="small-12 medium-3 columns">
-      <ul>
-        <li><b>Benefits</b></li>
-        <li><a href="/disability-benefits/">Disability Benefits</a></li>
-        <li><a href="/education/">Education &amp; Training</a></li>
-      </ul>
+      <dl>
+        <dt>Benefits</dt>
+        <dd><a href="/disability-benefits/">Disability Benefits</a></dd>
+        <dd><a href="/education/">Education &amp; Training</a></dd>
+      </dl>
     </div>
-    <div class="small-12 medium-4 columns">
-      <ul>
-        <li><b>Resources</b></li>
-        <li><a class="vets-app" href="/veterans-employment-center/">Veterans Employment Center&trade;</a></li>
-        <li><a class="vets-app" href="/facility-locator">Facility Locator</a></li>
-        <li><a class="vets-app" href="/gi-bill-comparison-tool/">GI Bill Comparison Tool</a></li>
-      </ul>
+    <div class="small-12 medium-3 columns">
+      <dl>
+        <dt>Resources</dt>
+        <dd><a class="vets-app" href="/veterans-employment-center/">Veterans Employment Center&trade;</a></dd>
+        <dd><a class="vets-app" href="/facility-locator">Facility Locator</a></dd>
+        <dd><a class="vets-app" href="/gi-bill-comparison-tool/">GI Bill Comparison Tool</a></dd>
+      </dl>
     </div>
     <!--Use with Signup <div class="small-12 medium-2 columns"> -->
     <div class="small-12 medium-3 columns">
-      <ul>
-        <li><b>Creating Vets.gov</b></li>
-        <li><a href="/playbook/">Playbook</a></li>
-        <li><a href="/experience/">Blog</a></li>
-      </ul>
+      <dl>
+        <dt>Creating Vets.gov</dt>
+        <dd><a href="/playbook/">Playbook</a></dd>
+        <dd><a href="/experience/">Blog</a></dd>
+        <dd><a href="/2015/11/11/why-we-are-designing-in-beta.html">Message from the Secretary</a></dd>
+      </dl>
     </div>
-    <div class="small-12 medium-2 columns">
-      <ul>
-        <li><b>Connect</b></li>
-        <li>
-          <div class="usa-social-links">
-            <a target="_blank" href="https://www.facebook.com/VeteransAffairs">
-              <svg width="26" height="39" role="img" aria-label="Facebook">
-                <title>Facebook</title>
-                <image xlink:href="<%= image_path('social-icons/svg/facebook25.svg')%>" src="<%= image_path('social-icons/png/facebook25.png')%>" width="26" height="39" />
```

Comment:
Yes, because they've been replaced by background images. _However_ I do need to move those images to this repo. 

This is probably something worth revisiting though. We should use SVG where possible.